sierra_adapter/sierra_to_dynamo/src/main/scala/uk/ac/wellcome/platform/sierra_to_dynamo/sink/SierraDynamoSink.scala

```diff
@@ -0,0 +1,37 @@
+package uk.ac.wellcome.platform.sierra_to_dynamo.sink
+
+import java.time.Instant
+
+import akka.Done
+import akka.stream.scaladsl.Sink
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB
+import com.gu.scanamo.{Scanamo, Table}
+import com.gu.scanamo.syntax._
+import io.circe.Json
+import io.circe.optics.JsonPath
+import uk.ac.wellcome.platform.sierra_to_dynamo.models.SierraRecord
+import uk.ac.wellcome.platform.sierra_to_dynamo.models.SierraRecord._
+
+import scala.concurrent.Future
+
+object SierraDynamoSink {
+  def apply(client: AmazonDynamoDB,
+            tableName: String): Sink[Json, Future[Done]] =
+    Sink.foreach(json => {
+      val record = SierraRecord(
+        id = JsonPath.root.id.string.getOption(json).get,
+        data = json.noSpaces,
+        updatedDate =
+          Instant.parse(JsonPath.root.updatedDate.string.getOption(json).get)
```

Comment:
Would having an `apply()` method on SierraRecord that takes a String as its third argument be useful?